PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -533,8 +533,8 @@ extension HomeViewController {
         self.present(activityVC, animated: true)
     }
 
-    private func present(url: URL?) {
-        guard true, let url = url else { return }
+    private func present(url: String?) {
+        guard let string = url, let url = URL(percentEncoding: string) else { return }
```

Comment:
maybe a bit more clarity with the names could help here? We have a parameter `url` which is a `String`, and a property `url` which is a `URL`